src/adv.py

```diff
@@ -49,3 +50,63 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+while True:
+
+    print(Room.__str__(player.current_room))
+    print("this is current room items", player.current_room.items)
+    print("this is players items", player.inventory)
+
+    action = input("Input an action: ")
+    if action[0] == "q":
+        exit()
+        print("Bye!")
+
+    elif action[0] == "n":
+        if player.current_room.n_to == None:
+            print("\nNothin' over there. Pick a new direction")
+        else:
+            player.moveRoom(player.current_room.n_to)
+    
+    elif action[0] == "e":
+        if player.current_room.e_to == None:
+            print("\nNothin' over there. Pick a new direction")
+        else:
+            player.moveRoom(player.current_room.e_to)
+
+    elif action[0] == "s":
+        if player.current_room.s_to == None:
+            print("\nNothin' over there. Pick a new direction")
+        else:
+            player.moveRoom(player.current_room.s_to)
+
+    elif action[0] == "w":
+        if player.current_room.w_to == None:
+            print("\nNothin' over there. Pick a new direction")
+        else:
+            player.moveRoom(player.current_room.w_to)
+
+    elif action == "take":
```

Comment:
Nice work Cory! When you refactor, look for where you're repeating yourself. Can you change your moveRoom method to implement some of the logic?